File: CodeForces/61A.cpp
#include <iostream>
using namespace std;
#include <string>
int main()
{
    string a, b, result;

    cin >> a;
    cin >> b;

    if (a.length() != b.length())
        return 1;

    for (int i = 0; i < a.length(); i++)
    {
        if (a[i] != b[i])
            cout << '1';
        else
            cout << '0';
    }

    return 0;
}

File: CodeForces/61A.py
def main():
    a = input()
    b = input()

    if len(a) != len(b):
        return 1

    result = ''
    for i in range(len(a)):
        result += '1' if a[i] != b[i] else '0'

    print(result)


if __name__ == '__main__':
    main()
